apps/pie-docs/src/components/icon-button/code/code.md

```diff
@@ -105,27 +107,33 @@ import '@justeattakeaway/pie-icon-button';
 
 ```html
 <!-- pass js file into <script> tag -->
-<pie-icon-button onclick="e => console.log(e)"><icon-close></icon-close></pie-icon-button>
+<pie-icon-button onclick="e => console.log(e)">
+  <icon-close></icon-close>
+</pie-icon-button>
 <script type="module" src="/main.js"></script>
 ```
 
-For Native JS Applications, Vue, Angular, Svelte etc.: 
+For Native JS Applications, Vue, Angular, Svelte, etc.:
 
 ```js
 // Vue templates (using Nuxt 3)
 import { PieIconButton } from '@justeattakeaway/pie-icon-button';
 
-<pie-icon-button @click="handleClick"><icon-close></icon-close></pie-icon-button>
+<pie-icon-button @click="handleClick">
+  <icon-close></icon-close>
+</pie-icon-button>
 ```
 
-For React Applications: 
+For React Applications:
 
-```js
+```jsx
```

Comment:
I could be mistaken but I expect you'd need to add the `jsx` support from `Prism` for this to work?

Reply:
It wouldn't have surprised me if that was the case, but the docs site preview seems to look okay!

| [Before](https://pie.design/components/icon-button/code/#changelog) | [After](https://pr1378-docs.pie.design/components/icon-button/code/#changelog) |
| --- | --- |
| ![2024-04-11 15_52_24](https://github.com/justeattakeaway/pie/assets/26894168/84d7a14d-d5fc-48b8-807f-dca9e46dbef9) | ![2024-04-11 15_53_10](https://github.com/justeattakeaway/pie/assets/26894168/a5c761cc-43dc-4f53-9c5a-63840b59c70a) |

Reply:
if it works it works I guess 😂